l10n-pocket/en/pocket/about.ftl

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev-pocket.allizom.org/about/
+
+pocket-about-us = About Us
+
+pocket-about-us-page-desc = { -brand-name-pocket } empowers people to keep their minds focused, stimulated, and nourished. We’re powered by a community of interesting people with curious minds.
+
+pocket-about-us-hi = Hi, we’re { -brand-name-pocket }.
+pocket-about-powered-by-community-of-interesting-people = We’re powered by a community of interesting people with curious minds, and we empower people to keep their minds focused, stimulated, and nourished. Which makes us a refreshing change from other digital spaces.
+pocket-about-us-save-inspirations = Save your inspirations
+
+# Variables:
+#   $savetopocket (url) link to https://getpocket.com/save-to-pocket
+#   $add_to_pocket (url) link to https://getpocket.com/add
+pocket-about-if-something-catches-your-interest = If something catches your interest, there’s probably something to it. With our <a id="save-button-for-the-internet" href="{ $savetopocket }"> save button for the internet</a> and our award-winning <a id="top-pocket-app" href="{ $add_to_pocket }">{ -brand-name-pocket } app</a> you can save anything from across the web in a distraction free space, for you to focus and let the ideas flow.
+
+pocket-about-find-hidden-gems = Find hidden gems
+
+pocket-about-internet-filled-with-buried-treasures = The internet is filled with buried treasures and our vibrant, well-read community has a real knack for finding them. Through the act of saving and consuming content, Pocket users signal where fascination can be found.
+
+# Variables:
+#   $pocket_home (url) link to https://getpocket.com/
+#   $pocket_home_domain (string) GetPocket.com
+#   $pocket_and_firefox (url) https://getpocket.com/pocket-and-firefox
+#   $pocket_daily_newsletter (url) https://getpocket.com/explore/pocket-hits-signup
+#   $add_to_pocket (url) link to https://getpocket.com/add
+#   $homepage_id (string) ID for HTML element
+#   $pocket_and_firefox_id (string) ID for HTML element
+#   $bottom_pocket_app_id (string) ID for HTML element
```

Comment:
Suggestion (non blocking): rather than making separate variables for each link attribute, it might make sense to define only one variable for each link e.g. `$pocket_home_attrs`. You can then pass a markup string for both the `id` and `href` attributes as a single variable. The advantage here is that we can freely add/remove attributes over time, without requiring translation changes or new strings IDs.

Example here:

- https://github.com/mozilla/bedrock/blob/main/l10n/en/firefox/browsers.ftl#L35-L38
- https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/templates/firefox/browsers/index.html#L137-L140

Same for other occurences below